docs/sources/analyze-results/correlate-results-in-grafana.md

```diff
@@ -72,11 +72,11 @@ Once added the k6 panel to a dashboard, you can edit the panel options.
 
 To change the panel visualization:
 
-1. Click on the panel title
+1. Hover over any part of the panel to display the actions menu on the top right corner. 
```

Comment:
```suggestion
1. Hover over any part of the panel to display the actions menu on the top right corner. 
```
Maybe we can even avoid saying top-right? That way we don't need to give any directional language that might change later.

Reply:
Leaving this with directional indicator since the menu to ensure the user finds it per our convo. Adding info about the description of the icon.